docs/UserGuide.adoc

```diff
@@ -160,7 +165,7 @@ Format: `clear`
 
 Exits the program. +
 Format: `exit`
-
+l
```

Comment:
There is the extra "l" which may have affected the presentation, and GitHub's ability to session the paragraph properly